Vostok.Logging.File/Rolling/Strategies/RollingStrategyHelper.cs

```diff
@@ -13,12 +13,12 @@ internal static class RollingStrategyHelper
 
         private static readonly char[] SuffixDashEliminators = {Path.DirectorySeparatorChar, Path.AltDirectorySeparatorChar, '.', '-'};
```

Comment:
No longer should be called dash eliminators since the separator might not be a dash anymore.